src/quo2/components/wallet/account_card/view.cljs

```diff
@@ -0,0 +1,70 @@
+(ns quo2.components.wallet.account-card.view
+  (:require [react-native.core :as rn]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.icon :as icon]
+            [quo2.components.wallet.account-card.style :as style]
+            [quo2.components.buttons.button.view :as button]
+            [utils.i18n :as i18n]
+            [quo2.components.markdown.text :as text]))
+
+(defn user-account-view
+  [{:keys [name balance percentage-value amount customization-color type emoji]}]
+  (let [watch-only? (= :watch-only type)]
+    [:<>
+     [rn/view (style/card customization-color watch-only?)
+      [rn/view style/profile-container
+       [rn/view {:style {:padding-bottom 2 :margin-right 2}}
+        [text/text {:style style/emoji} emoji]]
+       [rn/view {:style style/watch-only-container}
+        [text/text
+         {:size   :paragraph-2
+          :weight :medium
+          :style  (style/account-name watch-only?)}
+         name]
+        (when watch-only? [icon/icon :reveal {:color colors/neutral-50 :size 12}])]]
+
+      [text/text
+       {:size   :heading-2
+        :weight :semi-bold
+        :style  (style/account-value watch-only?)} balance]
+      [rn/view style/metrics-container
+       [rn/view {:style {:margin-right 4}}
+        [icon/icon :positive
+         {:color (if (and watch-only? (not (colors/dark?)))
+                   colors/neutral-50
+                   colors/white)
+          :size  16}]]
+       [text/text
+        {:weight :semi-bold
+         :size   :paragraph-2
+         :style  (style/metrics watch-only?)} percentage-value]
+       [rn/view (style/separator watch-only?)]
+       [text/text
+        {:weight :semi-bold
+         :size   :paragraph-2
+         :style  (style/metrics watch-only?)} amount]]]]))
+
+(defn add-account-view
```

Comment:
Please, make all vars private, except `view`. For `defn` we use `defn-` (not the metadata `^:private`).